### PR TITLE
Performance Profiler: General updates to CWV details area

### PR DESCRIPTION
**Pull request by gavande1**
## Proposed Changes

* Change user-friendly CWV names to standard CWV terms
* Update string for standard range bar
* Update font family for the badge

## Why are these changes being made?

* To match the new proposed design

## Testing Instructions

* Visit following URL on calypso live link. 
```
/speed-test-tool?url=https%3A%2F%2Fwww.atliq.com%2F&hash=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpZCI6NjkzNH0.uoPBbETEfzOEqTdJ_kro-zUpmvSF6vQz7JabJYumwME
````
- Verify that suggested changes are applied. 

## Pre-merge Checklist

- [ ] Has the general commit checklist been followed? (PCYsg-hS-p2)
- [ ] [Have you written new tests](https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md) for your changes?
- [ ] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [ ] Have you checked for TypeScript, React or other console errors?
- [ ] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [ ] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?
- [ ] For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-aUh-p2)?

### CODE REVIEW

File: client/performance-profiler/components/core-web-vitals-display/core-web-vitals-details.tsx [hunk 1]
```diff
@@ -109,7 +109,7 @@ export const CoreWebVitalsDetails: React.FC< CoreWebVitalsDetailsProps > = ( {
 					<div className="range">
 						<StatusIndicator speed="needsImprovement" />
 						<div className="range-description">
-							<div className="range-heading">{ translate( 'Moderate' ) }</div>
+							<div className="range-heading">{ translate( 'Need Improvement' ) }</div>
```
Comment (gcsecsey) [inline client/performance-profiler/components/core-web-vitals-display/core-web-vitals-details.tsx]:
```suggestion
							<div className="range-heading">{ translate( 'Needs Improvement' ) }</div>
```

File: client/performance-profiler/utils/metrics.ts [hunk 2]
```diff
@@ -3,88 +3,83 @@ import { Metrics } from 'calypso/data/site-profiler/types';
...
+			'First Contentful Paint reflects the time it takes to display the first text or image to visitors. The best sites load in under 1.8 seconds.'
 		),
 	},
 	lcp: {
-		good: translate( "Your site's largest content load is good" ),
-		needsImprovement: translate( "Your site's largest content load is moderate" ),
-		bad: translate( "Your site's largest content load needs improvement" ),
-		heading: translate( 'What is largest content load?' ),
-		aka: translate( '(aka Largest Contentful Paint)' ),
+		good: translate( "Your site's Largest Contentful Paint load is good" ),
```
Comment (gcsecsey) [inline client/performance-profiler/utils/metrics.ts]:
```suggestion
		good: translate( "Your site's Largest Contentful Paint is good" ),
```

File: client/performance-profiler/utils/metrics.ts [hunk 3]
```diff
@@ -3,88 +3,83 @@ import { Metrics } from 'calypso/data/site-profiler/types';
...
+		needsImprovement: translate( "Your site's Largest Contentful Paint load is moderate" ),
```
Comment (gcsecsey) [inline client/performance-profiler/utils/metrics.ts]:
```suggestion
		needsImprovement: translate( "Your site's Largest Contentful Paint is moderate" ),
```

File: client/performance-profiler/utils/metrics.ts [hunk 4]
```diff
@@ -3,88 +3,83 @@ import { Metrics } from 'calypso/data/site-profiler/types';
...
+		bad: translate( "Your site's Largest Contentful Paint load needs improvement" ),
```
Comment (gcsecsey) [inline client/performance-profiler/utils/metrics.ts]:
```suggestion
		bad: translate( "Your site's Largest Contentful Paint needs improvement" ),
```

File: client/performance-profiler/utils/metrics.ts [hunk 5]
```diff
@@ -3,88 +3,83 @@ import { Metrics } from 'calypso/data/site-profiler/types';
...
+		heading: translate( 'What is Largest Contentful Paint load?' ),
```
Comment (gcsecsey) [inline client/performance-profiler/utils/metrics.ts]:
```suggestion
		heading: translate( 'What is Largest Contentful Paint?' ),
```

File: client/performance-profiler/utils/metrics.ts [hunk 6]
```diff
@@ -3,88 +3,83 @@ import { Metrics } from 'calypso/data/site-profiler/types';
...
+		aka: translate( '(LCP)' ),
 		explanation: translate(
-			'Largest content load measures the time it takes for the largest visible element (like an image or text block) on a page to load. The best sites load in under 2.5 seconds.'
+			'Largest Contentful Paint load measures the time it takes for the largest visible element (like an image or text block) on a page to load. The best sites load in under 2.5 seconds.'
```
Comment (gcsecsey) [inline client/performance-profiler/utils/metrics.ts]:
```suggestion
			'Largest Contentful Paint measures the time it takes for the largest visible element (like an image or text block) on a page to load. The best sites load in under 2.5 seconds.'
```

**Review by gcsecsey — CHANGES_REQUESTED** [review]
I found some typos in the copy, we can merge this once all of these are updated.

**Review by gcsecsey — APPROVED** [review]
LGTM

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/16656484

Some locales (Hebrew, Japanese) have been temporarily machine-translated due to translator availability. All other translations are usually ready within a few days. Untranslated and machine-translated strings will be sent for translation next Monday and are expected to be completed by the following Friday.

Hi @gavande1, could you please edit the description of this PR and add a screenshot for our translators? Ideally it'd include all of the following strings:
- `Excellent`
- `Needs Improvement`
- `Your site's First Contentful Paint is good`
- `Your site's First Contentful Paint is moderate`
- `Your site's First Contentful Paint needs improvement`
- `What is First Contentful Paint?`
- `(FCP)`
- `Your site's Largest Contentful Paint is good`
- `What is Largest Contentful Paint?`
- `(LCP)`
- `Your site's Cumulative Layout Shift is good`
- `Your site's Cumulative Layout Shift is moderate`
- `Your site's Cumulative Layout Shift needs improvement`
- `What is Cumulative Layout Shift?`
- `(CLS)`
- `Your site's Interaction to Next Paint is good`
- `Your site's Interaction to Next Paint is moderate`
- `Your site's Interaction to Next Paint needs improvement`
- `What is Interaction to Next Paint?`
- `(INP)`
- `Your site's Time to First Byte is good`
- `Your site's Time to First Byte is moderate`
- `Your site's Time to First Byte needs improvement`
- `What is Time to First Byte?`
- `(TTFB)`
- `Your site's Total Blocking Time is good`
- `Your site's Total Blocking Time is moderate`
- `Your site's Total Blocking Time needs improvement`
- `What is Total Blocking Time?`
- `(TBT)`

Thank you in advance!

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/16656484).

**Pull request merged**